src/Microsoft.AspNet.Mvc.ViewFeatures/ViewComponents/DefaultViewComponentHelper.cs

```diff
@@ -15,12 +16,15 @@ public class DefaultViewComponentHelper : IViewComponentHelper, ICanHasViewConte
         private readonly IViewComponentDescriptorCollectionProvider _descriptorProvider;
         private readonly IViewComponentInvokerFactory _invokerFactory;
         private readonly IViewComponentSelector _selector;
+#pragma warning disable 0618
```

Comment:
Remove all of these pragmas. They were needed because the old type was `[Obsolete]`